app/src/main/java/org/mozilla/focus/browser/cursor/CursorView.kt

```diff
@@ -38,13 +34,12 @@ class CursorView(context: Context, attrs: AttributeSet) : View(context, attrs) {
     private val radius = context.resources.getDimensionPixelSize(R.dimen.remote_cursor_size) / 2f
     private val paint = Paint().apply {
         style = Paint.Style.FILL
-        alpha = CURSOR_ALPHA
         isAntiAlias = true
-        shader = RadialGradient(radius, radius, /* radius */ radius,
-                ContextCompat.getColor(context, R.color.teal50), ContextCompat.getColor(context, R.color.photonBlue50),
-                Shader.TileMode.CLAMP)
     }
 
+    private val cursorBg = BitmapFactory.decodeResource(resources, R.drawable.cursor_bg)
```

Comment:
These resources need to be in a -*hdpi folder, or else they won't get scaled (i.e. this will appear as different sizes on 720p and 1080p). Please put it in one of these folders (I'm not sure which one is appropriate to make this appear to be the same size – you'll have to calculate it from the multiplication scales: see the multipliers here: https://developer.android.com/training/multiscreen/screendensities.html#TaskProvideAltBmp)